18F/openFEC-web-app
# [WIP] Log client and server errors to Sentry.

Note: Not ready for merge; this is just for evaluating Sentry on CF!

This patch adds error reporting with Sentry. We get near-real-time reporting of both client and server errors, which will make debugging javascript much easier. We're also able to capture more information about the application state when the error was logged because the logs live on CF, not a third-party service.

To verify:
* Ask @dlapiduz for a Sentry account (I don't have permissions to create new accounts just now)
* Set the `SENTRY_DSN` and `SENTRY_PUBLIC_DSN` environment variables--you can can values from https://sentry.18f.gov/docs/platforms/python/ and https://sentry.18f.gov/docs/platforms/javascript/, respectively
* Introduce an error to the code
* Verify that errors are captured in Sentry

I'm so excited about this!

cc @LindsayYoung @dlapiduz
